src/views/authenticated/List.tsx

```diff
@@ -16,17 +19,47 @@ export function List({ data: unfilteredListItems }: Props) {
 		);
 	}, [searchTerm, unfilteredListItems]);
 
+	const handleAddItemsClick = () => {
+		navigate("/manage-list");
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
```

Comment:
Do we need this anymore?

I ask because I think this component would become a lot easier to read if we return early instead of inlining conditions inside the tsx.

Something like this:

```
if (unfilteredListItems.length === 0) {
  return ( 
     <> ... "your list is ready" </>
  )
}

return (
  <>... filter list, etc <>
)
```